packages/neon/neon/lib/src/widgets/cached_image.dart

```diff
@@ -27,58 +90,60 @@ abstract class NeonCachedImage extends StatefulWidget {
 }
 
 class _NeonCachedImageState extends State<NeonCachedImage> {
-  late Future<File> future = widget.getImageFile();
+  late Future<Uint8List> _future = widget.getImageFile();
```

Comment:
I'm not sure if it's good to have the underscore in stateful widgets. I tend to not have them because it's quite irrelevant, but I can also see why you still want to make it private. What do you think?

Reply:
At least it makes it a bit clearer that the scope is smaller and it's not a global variable (if we wouldn't have all the library thingies).

Reply:
The state is already 'private' so you are right.